docs/custom_domains.rst

```diff
@@ -1,85 +1,98 @@
-          If you want to setup a custom domain on `Read the Docs for Business <https://readthedocs.com/>`_,
-          please read our :doc:`commercial documentation <commercial/custom_domains>`.
+Once a project is imported into Read the Docs,
+by default it's hosted under a subdomain on one of our domains.
+If you need a custom domain, see :ref:`custom_domains:custom domain support`.
 
-Read the Docs supports a number of custom domains for your convenience. Shorter URLs make everyone happy, and we like making people happy!
+Subdomain support
+-----------------
 
-Subdomain Support
-------------------
+Every project has a subdomain that is available to serve its documentation.
+If you go to ``<slug>.readthedocs.io``, it should show you the latest version of your documentation.
+A good example is https://pip.readthedocs.io
+For :doc:`/commercial/index` the subdomain looks like ``<slug>.readthedocs-hosted.com``.
 
-Every project has a subdomain that is available to serve its documentation. If you go to ``<slug>.readthedocs.io``, it should show you the latest version of documentation. A good example is https://pip.readthedocs.io
+.. note::
 
-.. note:: If you have an old project that has an underscore (_) in the name, it will use a subdomain with a hyphen (-).
-          `RFC 1035 <https://tools.ietf.org/html/rfc1035>`_ has more information on valid subdomains.
+   If you have an old project that has an underscore (_) in the name,
+   it will use a subdomain with a hyphen (-).
+   `RFC 1035 <https://tools.ietf.org/html/rfc1035>`_ has more information on valid subdomains.
 
-Custom Domain Support
----------------------
 
-You can also host your documentation from your own domain by adding a domain to
-your project:
+Custom domain support
+---------------------
 
-* Add a CNAME record in your DNS that points the domain to: ``readthedocs.io``
-* Add a project domain in the :guilabel:`Domains` project admin page for your project.
+You can also host your documentation from your own domain.
 
 .. note::
-    We don't currently support pointing subdomains or naked domains to a project
-    using ``A`` records. It's best to point a subdomain, ``docs.example.com``
-    for example, using a CNAME record.
 
-Using pip as an example, https://pip.pypa.io resolves, but is hosted on our infrastructure.
+   We don't currently support pointing subdomains or naked domains to a project using ``A`` records.
+   It's best to point a subdomain, ``docs.example.com`` for example, using a CNAME record.
 
-As another example, fabric's dig record looks like this::
+.. tabs::
 
-    -> dig docs.fabfile.org
-    ...
-    ;; ANSWER SECTION:
-    docs.fabfile.org.   7200    IN  CNAME   readthedocs.io.
+   .. tab:: Read the Docs Community
+      
+      - Add a CNAME record in your DNS that points the domain to ``readthedocs.io``
+      - Go the :guilabel:`Admin` tab of your project
+      - Click on :guilabel:`Domains`
+      - Enter your domain and click on :guilabel:`Add`
```

Comment:
These ones should be numbered items since they are sorted steps (same for Read the Docs for Business tab)